networktables/entry.py

```diff
@@ -181,6 +181,18 @@ def getStringArray(self, defaultValue):
             return defaultValue
         return value[1]
     
+    @classmethod
+    def isValidDataType(cls, data):
+        return True
+        """
+        return (isinstance(data, int) or
```

Comment:
Also, this could be `isinstance(data, (str, int, bool))` ... it's probably faster?